apps/financial-aid/web-osk/src/components/Summary/AllFiles.tsx

```diff
@@ -31,7 +36,13 @@ const AllFiles = () => {
             if (file) {
               return (
                 <a
-                  href={file.name}
+                  onClick={() =>
+                    createSignedUrlMutation({
+                      variables: { input: { fileName: file.name } },
+                    }).then((response) => {
+                      window.open(response.data?.getSignedUrl.url, '_blank')
```

Comment:
window.open(response.data?.getSignedUrl.url, '_blank'), '_blank' is that not unneccary since this a tag has target="_blank"?